src/bg.c

```diff
@@ -928,79 +865,66 @@ void SetBgAffine(u8 bg, u32 srcCenterX, u32 srcCenterY, s16 dispCenterX, s16 dis
     SetBgAffineInternal(bg, srcCenterX, srcCenterY, dispCenterX, dispCenterY, scaleX, scaleY, rotationAngle);
 }
 
-u8 Unused_AdjustBgMosaic(u8 a1, u8 a2)
-{
-    u16 result;
-    s16 test1;
-    s16 test2;
-
-    result = GetGpuReg(REG_OFFSET_MOSAIC);
+#define BG_MOSAIC_SET 0
```

Comment:
forgot to mention previously... what style am I supposed to follow in this kind of situation? enum? and where to place it?

Reply:
either is fine, but it should probably be in a header. especially if it is something that could be used in other places.

Reply:
> either is fine, but it should probably be in a header (bg.h is fine). especially if it is something that could be used in other places.

Well, it's only referenced once (in poison field effect function, which was optimized in emerald to a direct IO buffer write). So should I put it in bg.h?

Reply:
yeah, I would